fair/forward.py

```diff
@@ -28,7 +28,109 @@ def iirf_interp_funct(alp_b,a,tau,iirf_h,targ_iirf):
     iirf_arr = alp_b*(np.sum(a*tau*(1.0 - np.exp(-iirf_h/(tau*alp_b)))))
     return iirf_arr - targ_iirf
 
+    
+def iirf_simple(c_acc, temp, r0, rc, rt, iirf_max):
+    """Simple linear iIRF relationship. Eq. (8) of Millar et al ACP (2017).
+    
+    Inputs:
+        c_acc    : cumulative airborne carbon anomaly (GtC) since
+                   pre-industrial
+        temp     : temperature anomaly since pre-industrial
+        r0       : pre-industrial time-integrated airborne fraction (yr)
+        rc       : sensitivity of time-integrated airborne fraction to airborne
+                   carbon (yr/GtC)
+        rt       : sensitivity of time-integrated airborne fraction to
```

Comment:
docstring for iirf_max?

Reply:
Well-spotted. New commit includes this.

I wonder if docstringing every single function is overkill. Probably only the ones that are intended to be external are necessary.

Reply:
Good practice is to add docstrings to all public functions. You have a lot of public functions so you might 'mark' a few more as private and then docstrings are more optional/for developer use (I say 'mark' as private as there's no such thing as private in Python really). 

As part of 2.0 refactoring let's talk more about some ways to tidy that sort of stuff up. For now it's just unnecessary extraneous noise so don't stress.